src/VirtoCommerce.Platform.Caching/Redis/RedisPlatformMemoryCache.cs

```diff
@@ -60,9 +103,6 @@ protected override void EvictionCallback(object key, object value, EvictionReaso
         {
             _log.LogInformation($"{nameof(RedisPlatformMemoryCache)}: channel[{_redisCachingOptions.ChannelName }] sending a message with key:{key} from instance:{ _cacheId } to all subscribers");
 
-            var message = new RedisCachingMessage { Id = _cacheId, CacheKeys = new[] { key } };
```

Comment:
Why you deleted this line? How the platform will inform the other instances about cache key eviction?

Reply:
Reverted back